data-pipeline/src/covid_variant_pipeline/assign_clades.py

```diff
@@ -129,14 +187,26 @@ def main(as_of_date: str):
 
     # incoming as_of_date comes in as a datetime object (for validation purposes), convert back to string now
     as_of_date = as_of_date.strftime("%Y-%m-%d")
-    logger.info("Starting pipeline", as_of_date=as_of_date)
+    now = datetime.datetime.now()
+    run_time = now.strftime("%Y%m%dT%H%M%S")
+
+    logger.info("Starting pipeline", as_of_date=as_of_date, run_time=run_time)
 
-    get_sequences()
-    get_sequence_metadata()
+    sequence_dir = get_sequences(run_time)
+    metadata_file = get_sequence_metadata(run_time, sequence_dir)
     reference_tree_path, root_sequence_path = save_reference_info(as_of_date)
-    assignment_file = assign_clades(as_of_date, reference_tree_path, root_sequence_path)
+    assignment_file = assign_clades(run_time, sequence_dir, reference_tree_path, root_sequence_path)
 
-    logger.info("Sequence clade assignments are ready", assignment_file=assignment_file)
+    merged_data = merge_metadata(as_of_date, now, metadata_file, assignment_file)
```

Comment:
This is returning every column available in the NCBI metadata and the output of the assignment CLI (see any *_clade_assignments.csv file in the data/assignment folder).

If it's useful, we can drop whatever columns aren't needed (or supply what we want in a list somewhere and use it to filter)